src/main/java/entity/Item.java

```diff
@@ -0,0 +1,50 @@
+package entity;
+
+public abstract class Item {
+    private final String name; // the name of the entity.Item
+    private int num; // the number of items in entity.Item
```

Comment:
I dont think we need this parameter in Items since Equipment objects don't need this (we will only have 1 weapon and 1 armor at all times). We can just have the name as the attribute, then move the count attribute to Collectibles since we only really need it for Collectible.

Reply:
If we remove the 'num' attribute remember to remove it as an argument in the constructor

Reply:
The num attribute for equipments could be greater than 1 if the player receives an equip they already have from monster. Then that extra equip can be sold for essence